### PR TITLE
New card layout and purchase flow (on LHS)

### PR DESCRIPTION
**Pull request by andygray**

### CODE REVIEW

File: src/components/pages/CompletePurchase.vue [hunk 1]
```diff
@@ -224,9 +185,15 @@
       },
       isNotSoldOut: function () {
         return this.edition.totalAvailable - this.edition.totalSupply > 0;
+      },
+      getArtistAddress: function (artist) {
```
Comment (jamesmorgan) [inline src/components/pages/CompletePurchase.vue]: Is it a good time to move this function to `artistUtils.js` or similar - I think its used in a few places

File: src/components/pages/Gallery.vue [hunk 2]
```diff
@@ -176,6 +133,8 @@
 
 
 <style scoped lang="scss">
+  @import '../../ko-colours.scss';
```
Comment (jamesmorgan) [inline src/components/pages/Gallery.vue]: I wonder if there is a way to define this globally, out of interest more than anything

File: src/components/ui-controls/cards/EditionCard.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="shadow-sm bg-white p-4">
+
+    <router-link
+      :to="{ name: 'artist-v2', params: { artistAccount: getArtistAddress(findArtistsForAddress(edition.artistAccount)) } }" class="artist-link">
```
Comment (jamesmorgan) [inline src/components/ui-controls/cards/EditionCard.vue]: This is a good argument for a better solution to the multiple address issue - `getArtistAddress(findArtistsForAddress(edition.artistAccount))`

File: src/components/ui-controls/cards/EditionCard.vue [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      USDPrice,
+      RarityIndicator
+    },
+    computed: {
+      ...mapGetters([
+        'findArtistsForAddress'
+      ])
+    },
+    methods: {
+      getArtistAddress: function (artist) {
```
Comment (jamesmorgan) [inline src/components/ui-controls/cards/EditionCard.vue]: duplicate function

File: src/components/pages/CompletePurchase.vue (hunk 1 above)
Comment (andygray) [reply src/components/pages/CompletePurchase.vue]: Agree. I will address in the clean-up

File: src/components/pages/Gallery.vue (hunk 2 above)
Comment (andygray) [reply src/components/pages/Gallery.vue]: Yeah. I did google once - I was originally copy and pasting it everywhere.

**Pull request merged**